recipes/simmate/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "simmate" %}
+{% set version = "0.0.0.dev3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # Once my github is made public, I can use archives from the releases with...
+  # url: https://github.com/jacksund/simmate/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: f5882ff5895550f50a70b8e798342faf5376fdd113cb47a6c4c5a10d916e775f
+
+build:
+  number: 0
+  skip: true  # [py<37]
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+      - simmate = simmate.command_line.base_command:simmate
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    # One of my dependencies (pymatgen --> spglib) requires this. Should I still
+    # include this compiler here?
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy ==1.21.4
+    - pandas ==1.3.4
+    - django ==3.2.9
+    - prefect ==0.15.10
+    - dask ==2021.11.2
+    - click ==8.0.3
+    - numba ==0.53.0
+    - django-crispy-forms ==1.13.0
+    - django-pandas ==0.6.1
+    - psycopg2-binary ==2.9.2
+    - dask-jobqueue ==0.7.3
+    - scikit-learn ==1.0.1
+    - dj-database-url ==0.5.0
+    - djangorestframework ==3.12.4
+    - django-filter ==21.1
+    - django-extensions ==3.1.5
+    - pyyaml ==6.0
+    - pytest ==6.2.5
+    - black ==21.11b1
+    - plotly ==5.4.0
+    - pymatgen ==2022.0.16
+    - pymatgen-analysis-diffusion ==2021.4.29
+
+test:
+  imports:
+    # only test a base import for now
+    - simmate
+  requires:
+    - pip
+  commands:
+    - pip check
+    - simmate --help
+
+about:
+  home: https://github.com/jacksund/simmate
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'Simmate is a toolbox for computational materials research.'
+  # Still setting things up, so these will be added later.
```

Comment:
Hm... I wonder if there is a policy about this... probably not.

Reply:
You mean making my repo public before giving it a BSD license? I can do that if you need.

Reply:
I just made it public if that helps 😄. 
https://github.com/jacksund/simmate